packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -416,7 +476,11 @@ class _TextSelectionHandleOverlayState extends State<_TextSelectionHandleOverlay
             new Positioned(
               left: point.dx,
               top: point.dy,
-              child: widget.selectionControls.buildHandle(context, type),
+              child: widget.selectionControls.buildHandle(
+                context,
+                type,
+                widget.renderObject.size.height / widget.renderObject.maxLines,
```

Comment:
Hi,
Seems this changes cause exception in case when TextField was created with unlimited lines `new TextField(maxLines: null)`. 
Steps:
* Create TextField with maxLines = null
* Enter some text, tap Done button
* Tap on TextField again

![200w_d](https://user-images.githubusercontent.com/3447934/28492534-bef717be-6f2f-11e7-8ad3-3e3fd23fe70f.gif)

```
➜  mobileapp git:(develop) ✗ flutter --version

Flutter • channel master • https://github.com/flutter/flutter.git
Framework • revision a92a62706c (16 hours ago) • 2017-07-21 16:57:33 -0700
Engine • revision 5fcfb995bb
Tools • Dart 1.25.0-dev.7.0
```

```
The following NoSuchMethodError was thrown building _TextSelectionHandleOverlay(dirty; state:
_TextSelectionHandleOverlayState#201d1()):
The method 'toDouble' was called on null.
Receiver: null
Tried calling: toDouble()

When the exception was thrown, this was the stack:
#0      Object._noSuchMethod (dart:core-patch/object_patch.dart:42)
#1      Object.noSuchMethod (dart:core-patch/object_patch.dart:46)
#2      double./ (dart:core-patch/double.dart:39)
#3      _TextSelectionHandleOverlayState.build (package:flutter/src/widgets/text_selection.dart:482:49)
#4      StatefulElement.build (package:flutter/src/widgets/framework.dart:3632:27)
#5      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3542:15)
#6      Element.rebuild (package:flutter/src/widgets/framework.dart:3443:5)
#7      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3522:5)
#8      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3660:22)
#9      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3517:5)
#10     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2901:14)
#11     Element.updateChild (package:flutter/src/widgets/framework.dart:2704:12)
#12     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4526:14)
#13     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2901:14)
#14     Element.updateChild (package:flutter/src/widgets/framework.dart:2704:12)
#15     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3554:16)
#16     Element.rebuild (package:flutter/src/widgets/framework.dart:3443:5)
#17     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3522:5)
#18     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3660:22)
#19     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3517:5)
#20     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2901:14)
#21     Element.updateChild (package:flutter/src/widgets/framework.dart:2704:12)
#22     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3554:16)
#23     Element.rebuild (package:flutter/src/widgets/framework.dart:3443:5)
#24     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3522:5)
#25     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3660:22)
#26     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3517:5)
#27     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2901:14)
#28     Element.updateChild (package:flutter/src/widgets/framework.dart:2704:12)
#29     RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:4317:32)
#30     MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:4641:17)
#31     Element.updateChild (package:flutter/src/widgets/framework.dart:2693:15)
#32     _TheatreElement.update (package:flutter/src/widgets/overlay.dart:505:16)
#33     Element.updateChild (package:flutter/src/widgets/framework.dart:2693:15)
#34     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3554:16)
#35     Element.rebuild (package:flutter/src/widgets/framework.dart:3443:5)
#36     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2279:33)
#37     BindingBase&SchedulerBinding&GestureBinding&ServicesBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:503:20)
#38     BindingBase&SchedulerBinding&GestureBinding&ServicesBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:189:5)
#39     BindingBase&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:688:15)
#40     BindingBase&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:636:9)
#41     _invoke (file:///Users/herman/Projects/github/flutter/bin/cache/pkg/sky_engine/lib/ui/hooks.dart:86)
#42     _drawFrame (file:///Users/herman/Projects/github/flutter/bin/cache/pkg/sky_engine/lib/ui/hooks.dart:75)
```